src/core/util/log.cc

```diff
@@ -123,6 +124,33 @@ static gpr_atm parse_log_severity(absl::string_view str, gpr_atm error_value) {
   return error_value;
 }
 
+void disable_vlog_for_grpc() { absl::SetVLogLevel("*grpc*/*", -1); }
+
+void gpr_to_absl_verbosity_setting_init(void) {
+  absl::string_view verbosity = grpc_core::ConfigVars::Get().Verbosity();
+  DVLOG(2) << "Log verbosity: " << verbosity;
+  if (absl::EqualsIgnoreCase(verbosity, "INFO")) {
+    LOG(WARNING) << "Not suitable for production. Prefer WARNING or ERROR. "
+                    "However if you see this message in a debug environmenmt "
+                    "or test environmenmt it is safe to ignore this message.";
+    absl::SetMinLogLevel(absl::LogSeverityAtLeast::kInfo);
+  } else if (absl::EqualsIgnoreCase(verbosity, "DEBUG")) {
+    LOG(ERROR) << "Not suitable for production. Prefer WARNING or ERROR. "
+                  "However if you see this message in a debug environmenmt or "
+                  "test environmenmt it is safe to ignore this message.";
+    absl::SetVLogLevel("*grpc*/*", 2);
```

Comment:
Why is the magic number 2? Lets say user uses VLOG(3), it doesn't get logged right? If there is any guideline to use debug statements less than or equal to 2, it makes sense. Is it there somewhere?

Reply:
gpr(DEBUG , ... ) corresponds to VLOG(2) . If anytone wants to alter it from 2 to something else, rather than using DEBUG, it would be better for them to set it via absl --v etc.